src/Global/EmoteStore.tsx

```diff
@@ -400,6 +402,12 @@ export namespace EmoteStore {
 				MainComponent.ShowTooltip.next({ event, hover: false });
 			});
 
+			inner.addEventListener('mousedown', _ => {
+				if(!settings.get('general.emote_click').asBoolean()) return;
```

Comment:
Use the configMap from SiteApp instead of settings for this to work.

Reply:
fixed